addons_marketplace/marketplace-manager/db/mongodb_client.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from flask_pymongo import PyMongo
+
+MONGO_URL = os.environ.get("ADDON_MARKETPLACE_MONGO_URL")
+MONGO_PORT = os.environ.get("ADDON_MARKETPLACE_MONGO_PORT")
+
+MONGO_BASE_ADDR = f"mongodb://{MONGO_URL}:{MONGO_PORT}"
+
+# contains available addons
+MONGO_ADDR_MARKETPLACE = f"{MONGO_BASE_ADDR}/marketplace"
+
+mongo_marketplace = None
+app = None
+
+
+def mongo_init(flask_app):
+    global app, mongo_marketplace
+
+    app = flask_app
+
+    mongo_marketplace = PyMongo(app, uri=MONGO_ADDR_MARKETPLACE).db["marketplace"]
```

Comment:
How exactly does this work?
If I am not mistaken there is no new MongoDB container running, will python spawn this on its own?

Reply:
you'd need to spin up the mongo container yourself if you want to use it, hence:
```
MONGO_URL = os.environ.get("ADDON_MARKETPLACE_MONGO_URL")
MONGO_PORT = os.environ.get("ADDON_MARKETPLACE_MONGO_PORT")
```

However I could make it more clear that for local development you'd need to spin up your own mongo db container.

Reply:
This would be helpful, because all other oakestra things can be spun up somehow by compose files so this would reduce misconseptions/confusion